x/perp/amm/keeper/bias_test.go

```diff
@@ -55,7 +55,7 @@ func TestBiasChangeOnMarket(t *testing.T) {
 			).
 			Then(
 				ammassertion.MarketShouldBeEqual(pairBtcUsdc,
-					ammassertion.Market_BiasShouldBeEqualTo(sdk.MustNewDecFromStr("9999.999900000001000000")), // Bias equal to PositionSize
+					ammassertion.Market_BiasShouldBeEqualTo(sdk.MustNewDecFromStr("-9999.999900000001000000")), // Bias equal to PositionSize
 				),
```

Comment:
These test cases are opposite. The trader opened long, so the bias should be positive. Semantically, we'd be saying the market has a long (positive) bias.